<?xml version="1.0" encoding="UTF-8"?>

<launch>

  <rosparam command="load" file="$(find eagleye_logger_driver)/config/eagleye_logger.yaml"/>
  <node pkg="eagleye_logger_driver" name="adis16475_node" type="adis16475_node" />

</launch>
